src/components/Content/SearchResult/MovieList/MovieList.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import MovieListCSS from './MovieList.module.css';
+import MovieCard from './MovieCard/MovieCard.jsx';
+
+const MovieList = (props) => (
+    <div className={MovieListCSS.ml_container}>
+        {props.arr.map((item,index) => {
+            return  <MovieCard key={index} elem={item}/>
+        })}
+    </div>
+)
+
+export default MovieList
```

Comment:
This component has props but doesn't have propTypes

Reply:
Fixed